src/app/store/preferences/preferences.effects.ts

```diff
@@ -28,6 +28,7 @@ export class PreferencesEffects {
   loadPreferences$ = createEffect(() => this.actions$.pipe(
     ofType(adminUiInitialized),
     mergeMap(() => {
+      this.store$.dispatch(clearPreferencesState());
```

Comment:
I think you can achieve the same by simply setting initial state on `adminUiInitialized`.